atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/TargetedSweeper.java

```diff
@@ -211,7 +229,8 @@ private void scheduleBackgroundThreads() {
                 executorService = PTExecutors
                         .newScheduledThreadPoolExecutor(numThreads, new NamedThreadFactory("Targeted Sweep", true));
                 for (int i = 0; i < numThreads; i++) {
-                    executorService.scheduleWithFixedDelay(this::runOneIteration, 1, 5, TimeUnit.SECONDS);
+                    executorService.scheduleWithFixedDelay(this::runOneIteration, 1000,
+                            Math.max(runtime.get().pauseMillis(), 1L), TimeUnit.MILLISECONDS);
```

Comment:
nit: this does enforce reasonability, though you might want to include a `Value.Check` in the immutable class itself (that `pauseMillis() > 0` or `>= 0` depending on taste)

Reply:
Yeah, I just don't want to error out if someone wants to disable pausing and puts in 0, which is quite a reasonable thing to do IMO.